frontend/src/components/GrantAccess.vue

```diff
@@ -4,26 +4,43 @@ import { ApiService, type ApplicationRoleResponse, type GrantUserRoleRequest } f
 import { selectedApplication } from '@/services/ApplicationState';
 import { onMounted, ref } from 'vue';
 import { useToast } from 'vue-toastification';
+import { Form as VeeForm, Field, ErrorMessage } from 'vee-validate';
+import router from '@/router';
+import * as yup from 'yup';
```

Comment:
Hey Nick, I guess it is possible to import just few things from 'yup' to reduce our bundle size, like this `import { object, string } from 'yup';
`?

Reply:
Nice suggestion! I have changed that.